src/products/utils/data.ts

```diff
@@ -124,16 +144,37 @@ export function getStockInputFromVariant(
 export function getVariantAttributeInputFromProduct(
   product: ProductVariantCreateData_product
 ): AttributeInput[] {
-  return product?.productType?.variantAttributes?.map(attribute => ({
-    data: {
-      inputType: attribute.inputType,
-      isRequired: attribute.valueRequired,
-      values: attribute.values
-    },
-    id: attribute.id,
-    label: attribute.name,
-    value: [""]
-  }));
+  const selectionAttributeInput = product?.productType?.selectionVariantAttributes?.map(
+    attribute => ({
+      data: {
+        inputType: attribute.inputType,
+        isRequired: attribute.valueRequired,
+        values: attribute.values,
+        variantAttributeScope: VariantAttributeScope.VARIANT_SELECTION
+      },
+      id: attribute.id,
+      label: attribute.name,
+      value: [""]
+    })
+  );
+
+  const notSelectionAttributeInput = product?.productType?.notSelectionVariantAttributes?.map(
```

Comment:
Same as above, since those two do exactly the same, would make sense to keep the logic in one place